tests/constants.py

```diff
@@ -24,7 +24,7 @@
 DOCKERFILE_ERROR_BUILD_PATH = os.path.join(FILES, 'docker-hello-world-error-build')
 DOCKERFILE_SUBDIR_PATH = os.path.join(FILES, 'df-in-subdir')
 
-FLATPAK_GIT = "git://pkgs.fedoraproject.org/modules/eog.git"
+FLATPAK_GIT = "http://pkgs.fedoraproject.org/modules/eog.git"
```

Comment:
```
$ git clone "git://pkgs.fedoraproject.org/modules/eog.git"
Cloning into 'eog'...
remote: Counting objects: 25, done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 25 (delta 8), reused 0 (delta 0)
Receiving objects: 100% (25/25), done.
Resolving deltas: 100% (8/8), done.
```

Maybe there were transient problems - the service wasn't running? If we're going to change this, probably better to use: `https://src.fedoraproject.org/modules/eog.git` to get HTTPS. (Compare what https://src.fedoraproject.org/modules/eog shows for GIT URLs.) In any case, the commit subject doesn't match the commit.

Reply:
Correct, I had some problems with Fedora Infra yesterday - I heard `git://` was disabled for outside hosts. Now it works fine, reverted that